lib/engine/ability/base.rb

```diff
@@ -9,14 +9,17 @@ class Base
       include Helper::Type
       include Ownable
 
+      attr_accessor :name, :value
       attr_reader :type, :owner_type, :remove, :when, :count
 
-      def initialize(type:, owner_type: nil, count: nil, remove: nil, **opts)
+      def initialize(type:, name: nil, owner_type: nil, count: nil, remove: nil, value: nil, **opts)
         @type = type&.to_sym
+        @name = name&.to_s || @type
         @owner_type = owner_type&.to_sym
         @when = opts.delete(:when)&.to_s
         @count = count
         @used = false
+        @value = value.nil? ? '' : value&.to_s
```

Comment:
ok a better name for value i think would be description because it's used for displaying text, value makes it seem like it has actual functional meaning

Reply:
Description could possibly be shown when clicking on the ability in the view.
I can rename it to display_value instead?